examples/nemo/scripts/acc_kernels_trans.py

```diff
@@ -77,9 +77,12 @@
         "tracers": {"variable": "jt"}
 })
 
-# List of all module names that PSyclone will chase during the creation of the
-# PSyIR tree in order to use the symbol information from those modules
-RESOLVE_IMPORTS = NEMO_MODULES_TO_IMPORT
+# Whether to chase the imported modules to improve symbol information (it can
```

Comment:
Same suggestions as in `user_scripts.rst`

Reply:
Done